### DISCUSSION TITLE
Lab 2 Assignment Clarification

### DISCUSSION THREAD

**Discussion by ghost** [albertkun/21S-ASIAAM-191A#54] [Help Desk] [answered]
Hi everyone!

I am just seeking some clarification regarding lab 2. I have completed everything in Lab2: Hello World and am at the bottom of the page looking at `style/style`. Are the requirements within this section for a completely new `index.html` where we build our Map Portfolio or are we just working with and adding to what we have completed within existing folders where we followed the other instructions?

I guess I want to know where I should be working and if we will be graded on two assignments (the in-class vs Map Portfolio) or if the in-class lab was just practice for the actual lab assignment.

Thank you!

**Comment by albertkun** [comment, marked as answer]
OH NO! I forgot to close that markdown. 😱 Thank you for catching it @ktgreenler!!! I just fixed it.

The in-class lab can be used as the actual lab assignment.  However, you are free to start new. As long as you fulfill the requirements listed for the Map Portfolio you are good. 


Let me know if you'd need more clarification.
